staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/request/admissionreview_test.go

```diff
@@ -0,0 +1,620 @@
+/*
+Copyright 2020 Authors of Arktos.
```

Comment:
Is this file auto generated or cherry-picked? If cherry-picked, seems missing Kubernetes copyright header.

Reply:
This is cherry-pick one. Missed in hack/arktos_copyright_copied_modified_k8s_files. Just added.